server/src/main/java/io/druid/metadata/IndexerSQLMetadataStorageCoordinator.java

```diff
@@ -835,11 +835,12 @@ private void updatePayload(final Handle handle, final DataSegment segment) throw
 
   public List<DataSegment> getUnusedSegmentsForInterval(final String dataSource, final Interval interval)
```

Comment:
Since we already have `getUnusedSegmentIntervals` in `SQLMetadataSegmentManager`, I think we should just move this method into `SQLMetadataSegmentManager` so all the segment table related methods are defined there.